terraform/modules/happy-env-ecs/integration_secret.tf

```diff
@@ -55,6 +55,7 @@ locals {
     }
 
     dynamo_locktable_name = aws_dynamodb_table.locks.id
+    datadog_api_key       = var.datadog_api_key
```

Comment:
hmm

Reply:
We can't reference czi-si secrets in this repo... Btw, I did create one.